core/src/program/blocks/join_block.rs

```diff
@@ -1,4 +1,10 @@
-use super::{fmt, hasher, Box, CodeBlock, Digest};
+use super::{fmt, hasher, Box, CodeBlock, Digest, Felt};
+
+// CONSTANTS
+// ================================================================================================
+
+/// Join domain used for hashing
+pub const JOIN_DOMAIN: Felt = Felt::new(91);
```

Comment:
You probably want to remove the TODO further down:

```rust
/// TODO: update hashing methodology to make it different from Split block.```

Similar with other TODOs around the code base about hashing.

Reply:
Good catch, I'll get that done.